website_sale_infinite_scroll/static/src/scss/main.scss

```diff
@@ -42,3 +42,25 @@ _:future,
         padding-bottom: 0.75rem !important;
     }
 }
+
+#html {
+    scroll-behavior: smooth;
+}
+
```

Comment:
**nitpick:** i would also add an hover selector just to make sure that the little underline doesn't come up

```css
.btn-back-to-top:hover {
    color: #133a39;
	text-decoration: none;
}
```

Before:
![immagine](https://github.com/OCA/e-commerce/assets/42804718/9c1acdb8-4b92-4bb8-b5f1-27ad5b142359)

After:
![immagine](https://github.com/OCA/e-commerce/assets/42804718/3db9fad4-0296-4b38-9a2d-1c80dff65996)

Reply:
Thanks a lot for the tips